### PR TITLE
Add SchemaVersion to version command output

### PR DESCRIPTION
**Pull request by spiffcs**
See attachment for new output and sample command updates

Screen Shot 2022-03-09 at 10 28 28 AM

Signed-off-by: Christopher Phillips <christopher.phillips@anchore.com

### CODE REVIEW

File: internal/version/build.go [hunk 1]
```diff
@@ -21,6 +23,7 @@ var platform = fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
 // Version defines the application version details (generally from build information)
 type Version struct {
 	Version        string `json:"version"`        // application semantic version
+	SchemaVersion  string `json:"schemaVersion"`  // application semantic JSON schema version
```
Comment (wagoodman) [inline internal/version/build.go:26]: nit: maybe `jsonSchemaVersion` in case we end up adding schemas for other formats in the future?

**Review by wagoodman — APPROVED** [review]
Thanks for the addition here --added a note on a rename, but shout out if you think that doesn't work and we can keep chatting 👍

**Comment by spiffcs** [comment]
> Thanks for the addition here --added a note on a rename, but shout out if you think that doesn't work and we can keep chatting 👍

Yep! Let me get the formatting fixed with that new rename and I'll push the changes here.

**Comment by spiffcs** [comment]
Screen Shot 2022-03-09 at 11 00 21 AM

I know it looks weird in the editor, but the spacing like that comes out correctly on the terminal

**Pull request merged**